postgres/test/__snapshots__/csn2postgres.test.js.snap

```diff
@@ -42,7 +42,7 @@ CREATE TABLE csw_TypeChecks (
   type_Time TIME,
   type_DateTime TIMESTAMP,
   type_Timestamp TIMESTAMP,
-  type_String VARCHAR(5000),
+  type_String VARCHAR(255),
```

Comment:
do we need a changelog entry for this?

Reply:
This comes from the compiler. So it is expected that most projects already have the latest compiler version. Which should have this as part of its changelog.